generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -426,6 +426,46 @@ class RingSystemBusRocketConfig extends Config(
   new freechips.rocketchip.system.BaseConfig)
 // DOC include end: RingSystemBusRocket
 
+class StreamingPassthroughRocketConfig extends Config(
+  new chipyard.example.WithStreamingPassthrough ++     // use top with tilelink-controlled passthrough
+  new chipyard.iobinders.WithUARTAdapter ++                      // display UART with a SimUARTAdapter
```

Comment:
Remove the comments here except for the Config fragment that is unique to this config